libraries/AP_HAL_ChibiOS/hwdef/ARKV6X/hwdef.dat

```diff
@@ -27,8 +27,11 @@ FLASH_SIZE_KB 2048
 
 env OPTIMIZE -O2
 
-# order of UARTs (and USB)
-SERIAL_ORDER OTG1 UART7 UART5 USART1 UART8 USART2 UART4 USART3 OTG2
+# order of UARTs (and USB) no IO MCU
+SERIAL_ORDER OTG1 UART7 UART5 USART1 UART8 USART2 UART4 USART6 USART3 OTG2
```

Comment:
This changes the serial port numbering for existing users, I think USART6 should come at the end to avoid that

Reply:
Thanks I'll move it to the back and force push.